props/prop_base.h

```diff
@@ -741,21 +741,22 @@ class PropBase : CommandParser, Looper, protected SaberBase {
 
   virtual void CheckLowBattery() {
     if (battery_monitor.low()) {
-      // TODO: FIXME
       if (current_style() && !current_style()->Charging()) {
-	LowBatteryOff();
-	
-	if (millis() - last_beep_ > 5000) {
+	if (SaberBase::IsOn()) {
+		LowBatteryOff();
+	} else if (millis() - last_beep_ > 15000) {  // (was 5000)
+		STDOUT.print(battery_monitor.battery());
```

Comment:
Maybe change this (and the following two lines) to

STDOUT << "Low battery: " << battery_monitor_.battery() << " volts\n";